Jenkinsfile

```diff
@@ -1274,14 +1274,55 @@ pipeline{
         				  }
                           success {
                             println("w12_SN_11-nightly Tests Completed")
-                            sendMail Email,'<h3>Tests Passed on Nigtly 10.0 ML Server on Windows Platform</h3><h4><a href=${RUN_DISPLAY_URL}>Check the Pipeline View</a></h4><h4> <a href=${BUILD_URL}/console> Check Console Output Here</a></h4>',false,'$BRANCH_NAME branch | Windows W2k12 | ML-11-Nightly | Single Node | Passed'
+                            sendMail Email,'<h3>Tests Passed on Nigtly 11.0 ML Server on Windows Platform</h3><h4><a href=${RUN_DISPLAY_URL}>Check the Pipeline View</a></h4><h4> <a href=${BUILD_URL}/console> Check Console Output Here</a></h4>',false,'$BRANCH_NAME branch | Windows W2k12 | ML-11-Nightly | Single Node | Passed'
                            }
                            unstable {
                               println("w12_SN_11-nightly Tests Failed")
-                              sendMail Email,'<h3>Some Tests Failed on Nightly 10.0 ML Server on Windows Platform </h3><h4><a href=${JENKINS_URL}/blue/organizations/jenkins/Datahub_CI/detail/$JOB_BASE_NAME/$BUILD_ID/tests><font color=red>Check the Test Report</font></a></h4><h4><a href=${RUN_DISPLAY_URL}>Check the Pipeline View</a></h4><h4> <a href=${BUILD_URL}/console> Check Console Output Here</a></h4><h4>Please create bugs for the failed regressions and fix them</h4>',false,'$BRANCH_NAME branch | Windows W2k12 | ML-11-Nightly | Single Node | Failed'
+                              sendMail Email,'<h3>Some Tests Failed on Nightly 11.0 ML Server on Windows Platform </h3><h4><a href=${JENKINS_URL}/blue/organizations/jenkins/Datahub_CI/detail/$JOB_BASE_NAME/$BUILD_ID/tests><font color=red>Check the Test Report</font></a></h4><h4><a href=${RUN_DISPLAY_URL}>Check the Pipeline View</a></h4><h4> <a href=${BUILD_URL}/console> Check Console Output Here</a></h4><h4>Please create bugs for the failed regressions and fix them</h4>',false,'$BRANCH_NAME branch | Windows W2k12 | ML-11-Nightly | Single Node | Failed'
                           }
                           }
         		}
+                stage('w12_cluster_11.0-2'){
+        			agent { label 'dhfWinCluster'}
```

Comment:
we have only one windows cluster. we cannot run multiple stages in parallel. you can run on a single node but not in cluster mode.